parent/build.gradle

```diff
@@ -125,7 +125,6 @@ ext.extraPomInfo = {
                     artifactId "exec-maven-plugin"
                     delegate.version '${exec-maven-plugin.version}'
                     delegate.configuration {
-                        executable("java")
```

Comment:
The default value for `exec.executable` property has to be defined for the application's POM to work out-of-the box. Maven does not specify any default for it (exec plugin seems to be designed as generic, although it falls back on `jdk` toolchain ;)), so if only java is omitted from exec plugin's config, the `mvn exec:exec` will fail with an error. 

Please see the pom snippet in #5020.

Reply:
The idea was that whoever wants to use `exec:exec`, needs to pass `-Dexec.executable=/path/to/java`.

We can also add a `<property>`.

Reply:
Understood, I would vote in favour of adding the `<property>` to keep the user experience of working with the Micronaut projects, both current and the newly generated: if both proposed changes are made, current projects (even with the newer parent POM) will continue to work 'as they are'. The Micronaut Launch web wizard also does not specify exec.executable in the project's pom.

Reply:
Done.